<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtrlSceneDialog</class>
 <widget class="QDialog" name="CtrlSceneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Pen</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QLabel" name="label_penWidth">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Pen Width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSlider_penWidth">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
        <property name="pageStep">
         <number>3</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pen Color:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="toolButton_penColor">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Brush</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Brush Color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButton_brushColor">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Shape</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox_shape">
        <item>
         <property name="text">
          <string>No shape</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horizontal line</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vertical line</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Any direction line</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rectangle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ellipse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Path</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
